build/presubmit.yaml

```diff
@@ -23,7 +23,7 @@ steps:
   args: ["build/md_check.sh"]
   id: Markdownlint check
 
-- name: "gcr.io/cloud-builders/go"
+- name: "gcr.io/cloud-foundation-cicd/cft/developer-tools@sha256:a09ad46f513847d31e2cfe20f818c2d64809e7d12695e2e6098900296f6527bf"
```

Comment:
We can use a newer one here 1dc28cd8c9a01a0a85330e2292035a6661c5adf19917fe1dff0ac43f66397693

Reply:
Done, also updated the templates